.github/scripts/clean-clusters.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+if [ -z "${IS_CI}" ]; then
+  echo "Error: This script is intended to run only in CI environments."
+  echo "Running it in your account may have destructive effects."
+  echo "Set the IS_CI environment variable to run this script."
+  exit 1
+fi
+
+REGIONS=("us-east-1" "us-east-2")
```

Comment:
I don't think we need to change this right now, but I think we should probably move CRUD regions to Github secrets/variables to avoid duplication between this file and the examples themselves.

Reply:
Agreed. Won't touch this now but it should be easy to pull these from an environment variable instead.

Reply:
Created #146